src/Commands/GenerateEventCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Nwidart\Modules\Commands;
+
+use Nwidart\Modules\Commands\GeneratorCommand;
+use Nwidart\Modules\Traits\ModuleCommandTrait;
+use Pingpong\Support\Stub;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+
+class GenerateEventCommand extends GeneratorCommand
+{
+	use ModuleCommandTrait;
+
+	protected $argumentName = 'name';
+	
+	/**
+	 * The console command name.
+	 *
+	 * @var string
+	 */
+	protected $name = 'module:make-event';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Generate a new Event Class for the specified module';
+
+	/**
+	 * Create a new command instance.
+	 *
+	 * @return void
+	 */
+	public function __construct()
+	{
+		parent::__construct();
```

Comment:
No need for a construct here since it doesn't do anything but calling the parent.